releasenotes/notes/whitelist-psycopg_binary.libs-3087e9e648a797a5.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Adds `psycopg_binary.libs` to the whitelist for the `datadog-agent-integrations-py3` package.
```

Comment:
I don't think you actually need a changelog here. It does not change much from a customer perspective and the bump to psycopg3 will be mentioned in the integration's changelog, wdyt?